common/src/objects/workload_state.rs

```diff
@@ -96,21 +100,21 @@ impl Display for StoppingSubstate {
         match self {
             StoppingSubstate::Stopping => write!(f, "Stopping"),
             StoppingSubstate::WaitingToStop => write!(f, "WaitingToStop"),
+            StoppingSubstate::DeleteFailed => write!(f, "DeleteFailed"),
         }
     }
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub enum SucceededSubstate {
     Ok = 0,
-    DeleteFailed = 8,
 }
 
 impl From<i32> for SucceededSubstate {
     fn from(x: i32) -> Self {
         match x {
             x if x == SucceededSubstate::Ok as i32 => SucceededSubstate::Ok,
-            _ => SucceededSubstate::DeleteFailed,
+            _ => SucceededSubstate::Ok,
```

Comment:
Do we need the 'match' at all, as in any case the return value is the same.

Reply:
Done.